packages/assets-controllers/src/Standards/ERC20Standard.ts

```diff
@@ -98,7 +98,15 @@ export class ERC20Standard {
 
     // Parse as bytes - treat empty string as failure
     try {
-      const utf8 = toUtf8(result);
+      // Not done in bytesToUtf8 in ethereumjs/util.
```

Comment:
WDYT about this (just to add a bit more context):

```suggestion
      // Not done in bytesToUtf8 in ethereumjs/util.
      // NOTE: It used to be trimmed in toUtf8 before, see:
      // https://github.com/ethereumjs/ethereumjs-monorepo/blob/%40ethereumjs/util%408.1.0/packages/util/src/bytes.ts#L300-L305
```